service/history/mutableStateBuilder.go

```diff
@@ -58,6 +58,10 @@ type (
 		updateRequestCancelInfos    []*persistence.RequestCancelInfo         // Modified RequestCancel Infos since last update
 		deleteRequestCancelInfo     *int64                                   // Deleted RequestCancel Info since last update
 
+		persistedBufferedEvents []*persistence.SerializedHistoryEventBatch // buffered history events that are already persisted
```

Comment:
Can we use the same naming convention like for other mutable state items. 
bufferedEvents and updateBufferedEvents

Reply:
will update